fxa-client/src/lib.rs

```diff
@@ -229,54 +228,29 @@ impl FirefoxAccount {
         self.state.login_state = state_machine.advance(state);
     }
 
-    fn oauth_cache_store(&mut self, info: &OAuthInfo) {
-        let info = info.clone();
-        let scope_key = info.scopes.join(" ");
-        self.state.oauth_cache.insert(scope_key, info);
-    }
-
-    fn scope_implies_scopes(scope: &str, match_scopes: &[&str]) -> Result<bool> {
-        let available_scopes = oauth::Scope::from_string(scope)?;
-        for scope in match_scopes {
-            let scope = oauth::Scope::from_string(scope)?;
-            if !available_scopes.implies(&scope) {
-                return Ok(false);
-            }
+    pub fn get_oauth_token(&mut self, scope: &str) -> Result<Option<OAuthInfo>> {
```

Comment:
do we have tests for the change where keys are now returned here for a particular scope? I might have missed it in the change